schema/db_migration.sql

```diff
@@ -0,0 +1,212 @@
+-- ids:
+--  id: bigserial
+--  idx: bytea # 16 bytes, indexed, used for lookup
+--  remainder: bytea # remainder of binary data
+--  type: smallint # 0=eoa 1=contract 2=txhash 3=blockhash
+-- TAMCHAU:
+--   "type" is postgres internal key, should use other name -> "id_type".
+--   Should ENUM instead of smallint for better performance (don't need to convert int to value later).
+--   Consider to add "created_at" column (maybe index for it also) in this table to track when a new
```

Comment:
it depends on application usecase, for this kind of improvement/suggestion, I'd recommend create new PR for it